### ISSUE TITLE
No value associated with key CustomCodingKeys(stringValue: "inline_traces", intValue: nil)

### ISSUE THREAD

**Issue by mschonvogel** [EOSIO/eosio-swift#218]
When calling `getActions` on `EosioRpcProvider` it returns an error: 
```
"RpcProviderFatalError: Error occurred in decoding/serializing returned data. KeyNotFound: inline_traces No value associated with key CustomCodingKeys(stringValue: \"inline_traces\", intValue: nil) (\"inline_traces\").."
``` 
#219

**Comment by mschonvogel** [comment]
https://github.com/EOSIO/eosio-swift/pull/219

**Comment by paulhyunkim** [comment]
Thank you for reporting this issue. This has been resolved by #226
